server/enterprise/metrics/metrics.go

```diff
@@ -1729,28 +1730,42 @@ func (mi *MetricsInterfaceImpl) SetReplicaLagTime(node string, value float64) {
 	mi.DbReplicaLagGaugeTime.With(prometheus.Labels{"node": node}).Set(value)
 }
 
-func (mi *MetricsInterfaceImpl) IncrementNotificationCounter(notificationType model.NotificationType) {
-	mi.NotificationTotalCounters.With(prometheus.Labels{"type": string(notificationType)}).Inc()
+func normalizeNotificationPlatform(platform string) string {
+	fmt.Printf("================================= %s\n", platform)
+	switch platform {
+	case "apple_rn-v2":
+		return "ios"
+	case "apple_rnbeta-v2":
+		return "ios_beta"
```

Comment:
Are you sure we should be differentiating the two versions of the iOS app? I'd sort of expect them to be grouped together with the non-beta build for this since there shouldn't be meaningful differences.

In other words, do you think it is that we'll want to see these separately or will all of our queries be written with `platform == 'ios' || platform == 'ios_beta'`?

Reply:
Since we have a difference, I thought to keep it, but you are right that it doesn't give us much relevant information (for now). Let's keep the cardinality low by just normalizing it to iOS.

Reply:
I'm not so familiar with clients but this information could be very useful only if we had the ios version as well. With this we could identify errors introduced with newer versions. But adding versions into play would bring plenty of unnecessary labels.

Reply:
After looking in other parts of the code, not all paths return the platform the same way (the ack in particular returns it as just "ios" or "android"). So let's keep it normalized.

Regarding versions, yes, the cardinality would increase more than needed. Specially since, in that scenario, we won't add any normalization on the server side (the server cannot reliably know which version is the latest, so any version number should be fine).